module/Decision/src/Decision/Controller/FileBrowser/FileNode.php

```diff
@@ -36,6 +78,12 @@ public function __construct($kind, $fullPath, $name)
         $this->kind = $kind;
         $this->fullPath = $fullPath;
         $this->name = $name;
+        if ($kind==='dir') {
```

Comment:
There should be spaces between the `===` and both operands.

Reply:
Resolved in 12c64c58ce22698636fce588f88fc82c928d41f9